migration/config/spire.spiffe.io_clusterspiffeids.yaml

```diff
@@ -0,0 +1,221 @@
+---
```

Comment:
It's a style thing, but I don't like to have extraneous separators when there's only one K8s object in the YAML file.

Reply:
removed